orm/model.go

```diff
@@ -0,0 +1,5 @@
+package orm
+
+func (m VersionedIDRef) NextVersion() VersionedIDRef {
+	return VersionedIDRef{ID: m.ID, Version: m.Version + 1}
```

Comment:
Okay, so this is where the version updates. I could really use better comments or docs